core_impl/src/main/java/io/opencensus/stats/MeasureToViewMap.java

```diff
@@ -84,47 +84,52 @@ synchronized void registerView(View view, Clock clock) {
   }
 
   // Records stats with a set of tags.
-  synchronized void record(StatsContextImpl tags, MeasureMap stats) {
+  synchronized void record(StatsContextImpl tags, MeasureMap stats, Clock clock) {
```

Comment:
The actual `clock` is stored in `StatsManager`: https://github.com/census-instrumentation/opencensus-java/blob/master/core_impl/src/main/java/io/opencensus/stats/StatsManager.java#L25. 
Call stack to this is: 
TagContext.record() -> StatsRecorded.record() -> StatsManager.record() -> EventQueue.enqueue() -> going through event queue -> EventQueue.process() -> MeasureToViewMap.record() -> MutableViewData.record(). 
In the last call we need to record the current `Timestamp` by calling `clock.now()`. Do you think it's better to get `Timestamp` here in MeasureToViewMap.record(), than passing in the `clock` reference and get `Timestamp` at MutableViewData.record()?